bittensor/core/async_subtensor.py

```diff
@@ -454,14 +454,14 @@ async def query_runtime_api(
 
     async def get_balance(
         self,
-        *addresses: str,
+        *addresses: tuple[str],
```

Comment:
```suggestion
        *addresses: str,
```
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values